dialogue/Story/Intro/IntroViolastro1.json

```diff
@@ -14,12 +14,17 @@
       "¿Pero que tenemos aquí? ¡Un grupito de aventureros!",
       "¡Estoy currandome las cosas para que esto sea más divertido para todos nosotros!\n¡Vuestro día a día necesita un poco más de chispa!"
     ],
+    "pl": [
+      "Patrzcie, oto nasi poszukiwacze przygód!",
```

Comment:
Given the context in this scene in which the "adventure searchers" are just in the temple, not too far from their town, you could use "bohaterowie", since the characters are taking the role of protectors rather than adventurers.

Reply:
That kinda makes sense, gotta update the dialogue.